docs/src/operate/security/device-certificate.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Device Certificate Setting
+tags: [Operate, Security, Cloud]
+sidebar_position: 2
+---
+
+## Obtaining a Device Certificate
+
+Obtaining a Device Certificate is a quite involved process:
+- A private key and a certificate signing request has to be created on the device
+- The certificate signing request is sent to the Certificate Authority (CA) chosen by the tenant for its fleet of devices
+- The CA checks that the device issuing the signing request is actually allowed to connect the tenant
+- The CA signs the signing request sending back the resulting certificate to the device
+- The tenant adds the signing certificate of the CA to its trusted list of certificate
+
+There are numerous variations notably on the contract between the tenant and the CA,
+the chain of signing certificates and the checks to be performed before approving a signing request.
+However, the outcome of the certificate signing process is always the same.
+It consists in three files:
+- The device private key - generated by the device and which must be kept secret as this the proof of ownership
+- The device certificate - signed by the CA and shared by the device on-demand
```

Comment:
We should probably mention that for at least c8y the public device certificate actually needs the full certificate chain (starting from the signing certificate (e.g. whatever was uploaded to c8y as the "trusted certificate").

e.g. `demo-device-007.pem` is actually a concatentation of 

```sh
# Create a new cert chain, which contains both the signing and device public cert
cat signing-cert.pem > demo-device-007.chain.pem
cat demo-device-007.pem >> demo-device-007.chain.pem

# Configure the cert chain file as the main cert to be used by thin-edge.io
tedge config set device.cert_path /etc/mosquitto/certs/demo-device-007.chain.pem
```

Reply:
Added